.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -0,0 +1,84 @@
+name: 'Bug report'
+labels: ['bug', 'triage']
```

Comment:
The current label is `kind/bug` and there is no label for triage. We currently have labels that start with `status/`. Should we have a label like `status/needs-triage`?

Reply:
Is that necessary to have 'kind' and 'status'? Either someone is assigned and left a comment, or not. Are we going to process these labels in an automated way?

Reply:
Automatically assigning labels is useful when trying to filter the type of issues.
I think it's a good idea to reuse existing labels so I am gonna change from "bug" to "kind/status" but I am not sure 

Would it makes sense to replace `triage` by `[zube]: To Triage` ?